staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -365,6 +374,21 @@ func (d *DiscoveryClient) SwaggerSchema(version schema.GroupVersion) (*swagger.A
 	return &schema, nil
 }
 
+// OpenAPISchema fetches the open api schema using a rest client and parses the json.
+// Warning: this is very expensive (~1.2s)
+func (d *DiscoveryClient) OpenAPISchema() (*spec.Swagger, error) {
+	data, err := d.restClient.Get().AbsPath("/swagger.json").Do().Raw()
```

Comment:
this is kind of off-topic. /swagger.json is not shown as an path when i visit the root of the server. It only shows "/swaggerapi/" which leads me to swagger-1.2. @mbohlool do you think we should fix that?